flask_mongoengine/__init__.py

```diff
@@ -44,7 +49,7 @@ def init_app(self, app):
 
         conn_settings = dict([
             (key.lower(), val)
-            for key, val in conn_settings.items()
+            for key, val in list(conn_settings.items())
```

Comment:
This is not necessary because you only want to iterate over the `conn_settings.items()` and not access items directly.